models/enterprise_sonic/evpn_esi_multihome/sonic_evpn_esi_multihome.yml

```
@@ -0,0 +1,47 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'community'
+    }
+NETWORK_OS: sonic
+RESOURCE: snmp
+COPYRIGHT: Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+DOCUMENTATION: |
+    module: sonic_evpn_esi_multihome
+    version_added: 4.5
+    short_description: Manage SNMP configuration on SONiC
+    description:
+      - This module provides configuration management of SNMP for devices running SONiC
+    author: Aida Shumburo (@aida-shumburo)
+    options:
+      config:
+        description:
+          - evpn esi multihome configuration
+        type: dict
+        suboptions:
+           mac-holdtime:
```

```suggestion
           mac_holdtime:
```
attribute names should use underscores and not hyphens